Description: Adds the missing plexus-component-metadata configuration
 that should be inherited from the parent plexus pom.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/plexus-java/pom.xml
+++ b/plexus-java/pom.xml
@@ -33,6 +33,20 @@
         <version>1.7.1</version>
         <!-- WATCH OUT duplicate compiler blocks confuses this plugin -->
         <!-- Generated components.xml copied to src/main/resources -->
+        <executions>
+          <execution>
+            <id>process-classes</id>
+            <goals>
+              <goal>generate-metadata</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>process-test-classes</id>
+            <goals>
+              <goal>generate-test-metadata</goal>
+            </goals>
+          </execution>
+        </executions>
         <configuration>
           <extractors>
             <extractor>source</extractor>
